app/assets/javascripts/views/searches/preview.js

```diff
@@ -216,8 +216,9 @@
       }
     }
 
-    table = tableBuilder.createTable(offset);
-    paginationData = tableBuilder.createPagination(offset);
+    var rows_per_page = json["rows_per_page"] || 25;
```

Comment:
When the property can be accessed via string in JS, it is usually better than using the squared brackets - I'll send a PR with an `.eslint` for this in the future.

```js
var rows_per_page = json.rows_per_page || 25;
```